Modules/DataModel/Sources/PocketCastsDataModel/Public/Bookmarks/Bookmark.swift

```diff
@@ -11,9 +11,23 @@ public struct Bookmark: Hashable {
 
     public let episodeUuid: String
     public let podcastUuid: String?
+
+    public static func == (lhs: Bookmark, rhs: Bookmark) -> Bool {
+        lhs.uuid == rhs.uuid
+    }
 }
 
 // MARK: - Identifiable
 extension Bookmark: Identifiable {
     public var id: String { uuid }
 }
+
+extension Bookmark {
+    public static let preview = Bookmark(uuid: "bookmark",
```

Comment:
Is this for SwiftUI previews? I wonder if we should extend `PreviewProvider` instead, given this is removed from the final build.

Reply:
Done here: 0bd77ac107d84e1eb5e22f91d6ec7471c3c6e6c5